src/components/badges/DiscordBadge.tsx

```diff
@@ -0,0 +1,17 @@
+/* @jsx MD */
+import MD from "jsx-md";
+import { Badge } from "../Badge";
+import type { BadgeComponent } from "./utils/BadgeComponent";
+
+/** Show a badge with the count of open issues on GitHub */
+export const DiscordBadge: BadgeComponent = ({ pkg }) => {
+  const { inviteLink, serverId } = pkg;
+  return (
+    <Badge
+      link={inviteLink || " "}
```

Comment:
Please do not add a fallback like this - a badge with an empty link does not make sense.